### ISSUE TITLE
[Bug] Keyboard navigation fails when Notes ItemsControl is hidden

### ISSUE THREAD

**Issue by SSchulze1989** [SSchulze1989/SimpleSudoku#1]
If the ItemsControl containing the CellNotes is hidden or collapes the navigation using the arrow keys does not work anymore.
